src/content/docs/en/configuring/config_toml.md

```
@@ -23,7 +23,7 @@ then paste it in here.
  -->
 
 ---
-HyDE exposes `xdg_config/hyde/config.toml` file for users to modify. This lets users have the ability to interact the scripts without using command arguments.
+Currently, HyDE exposes `~/.local/share/hyde/schema/config.toml` file for users to modify. This lets users have the ability to interact with the scripts without using command arguments.
```

markdownlint-cli2 (0.22.0)

[warning] Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)